sips/minutes/2016-10-25-sip-minutes.md

```diff
@@ -0,0 +1,173 @@
+Committee members:
+
+> The majority-plus-two rule sets a concrete threshold and fits the spirit of
+> the rule (not having simple majority, but a
+> [supermajority](https://en.wikipedia.org/wiki/Supermajority)). The 70% rule is
+> too strict, especially for committees that are not big, as ours. After some
+> research, we can choose the [two-thirds
+> rule](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) used in a
+> lot of political parties, parliaments, boards and committees.
+
+### Deciding the voting system
+
+Sébastien starts the discussion pointing out that he would like to have a simple
+percentage, without any rounding. Heather argues that this percentage changes
+depending on the number of the present Committee Members. Seth comments that
+it has been proposed in several online channels that all the Committee members
+should vote, even if they cannot attend the meeting. All the Committee agrees on
+this, but the main consequence is that the voting period is not predictable
+anymore, since there's not a fixed deadline to vote. Martin proposes to choose
+this fixed deadline case-by-case.
+
+Martin proposes that there's a meta rule to only accept a proposal if 50% of
+all the Committee members vote for it, and then to have another percentage on
+the actual vote. Seth, Eugene and Heather don't like this idea. Heather brings
+up the issue of abstentions, should they still be considered as no's even if
+people are on vacation or cannot vote?. Adriaan says this is a fair concern, and
+thinks that there should be a fixed deadline for voting.
+
+All the Committee agrees that with the new rules the Committee wouldn't vote
+publicly, because Committee members can notify their vote after the meeting and
+before the fixed deadline.
+
+Martin proposes to keep the old rules, the 70% percentage rule. Jorge proposes
+to have simpler and more predictable rules, and replace 70% with the two-thirds
+rule. Martin comes back to the abstentions point: he doesn't want them to be
+considered as no, because with the current rules Committee members could block
+the whole decision process. Some Committee members point out that this wouldn't
+happen because then the quorum wouldn't be reached. The discussion shifts
+towards the quorum point: is it necessary anymore? Heather and Adriaan say that
+it is, because it ensures that a reasonable number of Committee members meet to
+discuss proposals and convince each other about the pros and cons. All the
+Committee finally decides to not consider abstentions as no's. Adriaan, however,
+suggests that we should penalize people that abstain too often.
+
+Another system proposed by Martin helps survive the abstention issue mentioned
+before. For a proposal to pass, at least 50% of all the Committee 
+and two-thirds of all the people voting (not counting abstentions) must accept
+it. The quorum rule is kept. After some discussion, the Committee unanimously
+votes in favor of the new rules.
```

Comment:
It's not clear from these notes which alternative was accepted in the end. Some more detail should be devoted to explaining the voting rule in the minutes. So far, to my knowledge, this is the only place the rule has been written so far. 

Has the SIP process been updated with the new voting rule?